.lein-repl-history

```diff
@@ -0,0 +1,61 @@
+*ns*
```

Comment:
you could put this file in the .gitignore

Reply:
Done. Target files are just the ones with ".nameOfFile" right? Not target folder?